sdks/python/apache_beam/typehints/opcodes.py

```diff
@@ -327,7 +339,18 @@ def call_function(state, arg, has_var=False, has_kw=False):
 
 
 def make_function(state, arg):
-  state.stack[-arg - 1:] = [Any]  # a callable
+  """Creates a function with the arguments at the top of the stack.
+  """
+  # TODO(luke-zhu): Handle default argument types
+  globals = state.f.__globals__ # Inherits globals from the current frame
+  if sys.version_info[0] == 2:
+    func_code = state.stack[-1].value
+    func = types.FunctionType(func_code, globals)
+  else:
+    func_name = state.stack[-1].value
+    func_code = state.stack[-2].value
+    func = types.FunctionType(func_code, globals, name=func_name)
+  state.stack.append(Const(func))
```

Comment:
<!--new_thread; commit:b015963f671ceb39f8033014eb6d4e46a4a734bc; resolved:0-->
why `Const(func)`?

Reply:
This aligns it with the code in CALL_FUNCTION which only accepts functions wrapped by a Const. I believe this may be due to how user-defined functions are processed.